### DISCUSSION TITLE
Robust LC/intensity calculations

### DISCUSSION THREAD

**Discussion by lenawilde** [vdemichev/DiaNN#259] [Q&A]
Hi,

I have two questions concerning answers to reviewers :-).

we analyzed our dataset using DIA-NN in the Robust LC setting, as it is recommended in the manual, which we mentioned in the M&M section. Now a reviewer asked for more detailed explanation. I was looking for more info on that, but was not successful. Is the peak height used instead of the area under the peak? Maybe you could help me with a citation or a more detailed explanation?

and secondly:

Could you maybe comment, if my explanation of the intensity calculation and normalisation is explained like that in a good way?
"Peptide quantification was done by applying the LFQ algorithm on summarized MS2 fragment quantities. The LFQ normalized peptide quantities were summarized on the protein level by taking the median."

Many, many thanks in advance!
Lena

**Comment by vdemichev** [comment]
Hi Lena,

- LC performance can vary. Sometimes peak broadening can occur, that is in the beginning of an experiment peaks might be sharp, whereas in the end they might become less sharp. This effect, if present, will affect quantification. "Any LC" will partially alleviate this by adaptively choosing how many data points (around the peak apex) to integrate, depending on the observed peak width. DIA software does not integrate the whole peak - the aim is always to integrate only around the apex, otherwise lots of interferences would be integrated leading to poor accuracy. So the way Any LC works: if the peak has, say, 3 data points at >= half maximum, Any LC will likely integrate these 3 data points, while if the peak of the same peptide in a different run has 5 data points at >= half maximum (here I am exaggerating, that would be quite significant broadening), 5 will likely be integrated. With Robust LC the number of integrated data points is always fixed. So this method is best suited for experiments in which there's no peak broadening, or peak broadeining is insignificant. The advantage is that any automatic decision logic, including that used by Any LC, can be affected by noise in the data, and hence can produce noisy results, leading to a slight loss in precision (i.e. slightly higher CVs). Now, DIA-NN does report the average peak width as measured 'in scans' (i.e. DIA cycles) in its .stats.tsv report file (not to be confused with average number of data points per peak). This reported number does depend on the total injection amount (less injected = lower number), but if for comparable injection amounts with comparable ID numbers the variation in reported peak width is minimal, then no broadening occured and can use Robust LC. Even with mild peak broadening can often use Robust LC as batch effects due to mass spec performance migth be more significant anyway. We ourselves have never acquired a dataset where we would see peak broadening significant enough to warrant using Any LC. So the only reason it's set as the default option in DIA-NN is because in theory it's somewhat more generally applicable. So yes, for most experiments Robust LC is the better choice.

- I am not sure what intensities are you referring to. If it's DIA-NN's quantities from the output matrices or from one of the 'MaxLFQ' report columns, then better to just refer to the "MaxLFQ algorithm [REF J. Cox paper 2014] as implemented in DIA-NN" - no need to explain how this is calculated. It's not exactly median peptide intensity.

Hope this helps! Please let me know if there any other questions about DIA-NN.

Best,
Vadim

**Reply by lenawilde** [reply]
Dear Vadim,
you are the best! Thank you so much for this detailed explanation, it helps a lot.
We really appreciate your help!

Best,
Lena
